### DISCUSSION TITLE
How go get Bulk Customers and Bulk Orders from Adobe Commerce GraphQL API call

### DISCUSSION THREAD

**Discussion by hey-krishnendu** [AdobeDocs/commerce-webapi#84] [Q&A]
In GraphQL API docs, we can able to get a single customer as per API docs, But we need to know customer username and password, Is there any way we can get all the customers by Customer/search API call like we do in REST API. And We haven't find any way  to get the Orders.

Please provide a way to get Multiple Customers and Orders as we can get in REST Apis

**Comment by keharper** [comment]
@KrishSam874 If I understand your question, you're looking to replicate the abilities of `GET V1/customers/search` and `GET V1/orders` in GraphQL. 

 These use cases are not supported. Our GraphQL APIs are designed to transmit data between the backend and the storefront, enabling the shopper to perform any action on the storefront. Retrieving a list of customers or orders from multiple shoppers is a task that requires admin permissions. (I know I'd be upset if I learned that another customer has access to my account information or a list of my orders.) 

I'm not aware of any initiative to create a GraphQL Admin API, so you'll have to use REST for the foreseeable future.
